src/main/java/de/cismet/cids/custom/objecteditors/wunda_blau/QsgebMarkerEditor.java

```diff
@@ -0,0 +1,1595 @@
+            cbStatus.setEnabled(false);
+            cbErgebnis.setEnabled(false);
+            txtLage.setEnabled(false);
+            taBemerkung.setEnabled(false);
+            lblGeom_txt.setVisible(false);
+            jCkbHistorisch.setEnabled(false);
+        }
+    }
+    
+    private void setHistorischOnly() {
+        if ((cidsBean.getProperty(FIELD__HISTORISCH) == null) || Boolean.FALSE.equals(cidsBean.getProperty(FIELD__HISTORISCH))) {
+            lblHistorisch_txt.setVisible(false);
+            jCkbHistorisch.setVisible(false);
+            lblDatumHistorisch_txt.setVisible(false);
+            lblDatumHistorisch.setVisible(false);
+        }
+    }
+
+    /**
+     * DOCUMENT ME!
+     */
+    public void setMapWindow() {
+        final CidsBean cb = this.getCidsBean();
+        try {
+            if (cb.getProperty(FIELD__GEOREFERENZ) != null) {
+//panPreviewMap.initMap(cidsBean, "FIELD__GEOREFERENZ__GEO_FIELD", 50.0, "http://s10221.wuppertal-intra.de:7098/alkis/services?&VERSION=1.1.1&REQUEST=GetMap&FORMAT=image/png&TRANSPARENT=FALSE&BGCOLOR=0xF0F0F0&EXCEPTIONS=application/vnd.ogc.se_xml&LAYERS=alf&STYLES=&BBOX=<cismap:boundingBox>&WIDTH=<cismap:width>&HEIGHT=<cismap:height>&SRS=EPSG:25832");
+                panPreviewMap.initMap(cb, FIELD__GEOREFERENZ__GEO_FIELD, 20.0);
+            } else {
+                final int srid = CrsTransformer.extractSridFromCrs(CismapBroker.getInstance().getSrs().getCode());
+                final BoundingBox initialBoundingBox;
+                initialBoundingBox = CismapBroker.getInstance().getMappingComponent().getMappingModel()
+                            .getInitialBoundingBox();
+                final Point centerPoint = initialBoundingBox.getGeometry(srid).getCentroid();
+
+                final MetaClass geomMetaClass = ClassCacheMultiple.getMetaClass(
+                        CidsBeanSupport.DOMAIN_NAME,
+                        TABLE_GEOM,
+                        getConnectionContext());
+                final CidsBean newGeom = geomMetaClass.getEmptyInstance(getConnectionContext()).getBean();
+                newGeom.setProperty(FIELD__GEO_FIELD, centerPoint);
+                panPreviewMap.initMap(newGeom, FIELD__GEO_FIELD, 20.0);
+            }
+        } catch (final Exception ex) {
+            Exceptions.printStackTrace(ex);
+        }
+    }
+    
+    public StringBuilder setUserStatus(){
+        final StringBuilder errorMessage = new StringBuilder();
+        
+        if ((isEditor)) {
+            if (cidsBean.getProperty(FIELD__STATUS_ID) != null){
+                int status = (int)cidsBean.getProperty(FIELD__STATUS_ID);
+                DateFormat df = DateFormat.getDateInstance(DateFormat.MEDIUM);
+                Date datum = new Date();
+                try{
+                    switch (statusAttribute){
+                        case 0://Prüfen
+                            if (status == 1 || status == 4){ //--> Zur Bearbeitung || Keine Bearbeitung erforderlich
```

Comment:
Again: use the natural key here instead of the IDs (which are not guaranteed to never change) !